<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="xiaolanyun" name="管理员登陆" tests="1" failures="1" timestamp="16 Apr 2019 03:11:17 GMT" time="21.795" errors="0">
  <testcase name="checkUI" time="0.009" classname="com.xiaolanyun.appium.DataTest.testCase.Login.LoginPage_001_UICheckForRootLogin_Test">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.xiaolanyun.appium.DataTest.utils.AppiumUtil.findElement(AppiumUtil.java:92)
	at com.xiaolanyun.appium.DataTest.utils.AppiumUtil.typeContent(AppiumUtil.java:116)
	at com.xiaolanyun.appium.DataTest.pagesHelper.Mart_LoginPageHelper.clickForRootLogin(Mart_LoginPageHelper.java:42)
	at com.xiaolanyun.appium.DataTest.testCase.Login.LoginPage_001_UICheckForRootLogin_Test.checkUI(LoginPage_001_UICheckForRootLogin_Test.java:25)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- checkUI -->
</testsuite> <!-- 管理员登陆 -->
